examples/lib/defaultHtmlStepUi.js

```diff
@@ -191,7 +199,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
     function saveOptions(e) {
```

Comment:
Hey can you expand to see line no 188-190 this is some new code added and the new approach needs to be added for them as well! just make sure its done everywhere I remembered these areas but do check for new areas there have recently been a few merges!

Reply:
which new approach are you talking about @aashna27 ?

Reply:
Oh the UI. Ok ok I will update it

Reply:
done